NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -142,21 +165,26 @@ class NeuroID private constructor(
     }
 
     fun setEnvironment(environment: String) {
-        NIDServiceTracker.environment = environment
+        NIDLog.i(
+            "NeuroID Info",
+            "**** NOTE: setEnvironmentProduction METHOD IS DEPRECATED"
+        )
     }
 
     fun setEnvironmentProduction(prod: Boolean) {
-        if (prod) {
-            NIDServiceTracker.environment = "LIVE"
-        } else {
-            NIDServiceTracker.environment = "TEST"
-
-        }
+        NIDLog.i(
+            "NeuroID Info",
+            "**** NOTE: setEnvironmentProduction METHOD IS DEPRECATED"
+        )
     }
 
     fun getEnvironment(): String = NIDServiceTracker.environment
 
     fun setSiteId(siteId: String) {
+        NIDLog.i(
+            "NeuroID Info",
+            "**** NOTE: setSiteId METHOD IS DEPRECATED"
+        )
```

Comment:
Should we also have this log for the getSiteId() method since we won't have the site id set by the user going forward?

Reply:
We want a message that its deprecated but we aren't removing it completely yet so that when people upgrade it doesn't break

Reply:
I was asking about the _get_ siteId method. We are not using that method in the sdk as far as I can see. So does that also need to be deprecated?

Reply:
Oh I misread, good catch. Will add a message now